doc/architecture.rst

```diff
@@ -278,16 +278,17 @@ the client and the monitor share a secret key.
                 | transmit key  |
                 |               |
 
-To authenticate with the monitor, the client passes in the user name to the
-monitor, and the monitor generates a session key and encrypts it with the secret
-key associated to the user name. Then, the monitor transmits the encrypted
-ticket back to the client. The client then decrypts the payload with the shared
-secret key to retrieve the session key. The session key identifies the user for
-the current session. The client then requests a ticket on behalf of the user
-signed by the session key. The monitor generates a ticket, encrypts it with the
-user's secret key and transmits it back to the client. The client decrypts the
-ticket and uses it to sign requests to OSDs and metadata servers throughout the
-cluster.
+Here are the steps in the process whereby a client authenticates with a
```

Comment:
Seems a bit stilted.

Here are the steps when a client authenticates with a monitor.

Reply:
I don't like the adverb of time "when", so I'm going with the adverb of manner "how":

> Here is how a client authenticates with a monitor.

Maybe that will make it sound a bit less like it was written by Philboyd Studge.